sdk/schemaregistry/Azure.Data.SchemaRegistry/api/Azure.Data.SchemaRegistry.netstandard2.0.cs

```diff
@@ -45,12 +46,12 @@ public partial class SchemaRegistryClient
     }
     public partial class SchemaRegistryClientOptions : Azure.Core.ClientOptions
     {
-        public SchemaRegistryClientOptions(Azure.Data.SchemaRegistry.SchemaRegistryClientOptions.ServiceVersion version = Azure.Data.SchemaRegistry.SchemaRegistryClientOptions.ServiceVersion.V2023_07) { }
+        public SchemaRegistryClientOptions(Azure.Data.SchemaRegistry.SchemaRegistryClientOptions.ServiceVersion version = Azure.Data.SchemaRegistry.SchemaRegistryClientOptions.ServiceVersion.V2023_07_01) { }
         public enum ServiceVersion
         {
             V2021_10 = 1,
             V2022_10 = 2,
-            V2023_07 = 3,
+            V2023_07_01 = 3,
```

Comment:
This hasn't been released yet.

Reply:
Is this the protobuf one or just one that we meant to release for JSON?

Reply:
@jsquire it is the Protobuf one

Reply:
2022_10 is the JSON/Custom one and has been released but only in beta